artifact-management/commands/gem-test-common-steps.yml

```diff
@@ -0,0 +1,16 @@
+description: Perform common gem test steps
+parameters:
+  gem_test_dir:
+    description: The directory in which the gem tesets should run
+    type: string
+    default: .
+  bundler_version:
```

Comment:
This default should probably be sensitive to the version of Ruby being used. For Ruby 2.7 or later, Bundler 2.x is already included in the Ruby distribution and it is very hard to remove it.